cesium/featurize.py

```diff
@@ -29,7 +46,7 @@ def load_and_store_feature_data(features_path, output_path):
 def featurize_time_series(times, values, errors=None, features_to_use=[],
                           targets=None, meta_features={}, labels=None,
                           custom_script_path=None, custom_functions=None,
-                          use_celery=False):
+                          get=dask.multiprocessing.get):
```

Comment:
The only thing I might change is the keyword "get", which is not very descriptive.